config/crd/bases/orkestra.azure.microsoft.com_applicationgroups.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.9
+    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
Can you upgrade your kubebuilder binary to v0.2.9

Reply:
done